app/assets/javascripts/modules/help.js.coffee

```diff
@@ -61,30 +61,61 @@ ns = @edsc.models.page
       showNext: true
       },{
       title: "Facets"
-      content: "<p>Refine your search further with available facets, such as:
-                <div style='margin-left: 15px;'><ul style='list-style-type: disc;'>
-                  <li>Features - subsettable, near-real-time or has imagery</li>
+      content: '<p>Refine your search further with available facets, such as:
+                <div class="tour-indent"><ul class="bullet-list">
```

Comment:
Let's change this class from "tour-indent" to "tour-list". Remember, class names should describe content, not styles, as much as possible.